src/components/StabilityFeeAlert.jsx

```diff
@@ -0,0 +1,34 @@
+// Libraries
+import React from "react";
+import {inject, observer} from "mobx-react";
+
+// Components
+import InlineNotification from "./InlineNotification";
+
+@inject("content")
+@inject("network")
+@observer
+class StabilityFeeAlert extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = { show: true }
```

Comment:
based on https://github.com/makerdao/scd-cdp-portal/pull/31/files#r275381192

we don't need local state.